Makefile

```diff
@@ -4,7 +4,7 @@ VERSION?=latest
 MANAGER_CERT_PATH = "./certs/cluster-manager.crt.pem"
 LD_FLAGS = "-X github.com/kelda/blimp/pkg/version.Version=${VERSION} \
 	   -X github.com/kelda/blimp/cli/manager.ClusterManagerCertBase64=$(shell base64 ${MANAGER_CERT_PATH} | tr -d "\n") \
-	   -X github.com/kelda/blimp/cli/login.LoginProxyHost=${LOGIN_PROXY_HOSTNAME} \
+	   -X github.com/kelda/blimp/cli/login.LoginProxyGRPCHost=${LOGIN_PROXY_GRPC_HOSTNAME} \
```

Comment:
Should set `LOGIN_PROXY_GRPC_HOSTNAME` above as well right?

Reply:
ah yeah, had it set in local.mk so forgot about that.